conf/application.conf

```diff
@@ -47,7 +47,28 @@ include "validation-config/SAYE/ers-saye-exercised-validation.conf"
 application.global=playconfig.ApplicationGlobal
 
 security.headers.filter.enabled = true
-play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9250 localhost:9032 assets.digital.cabinet-office.gov.uk www.google-analytics.com data:"
+//play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9250 localhost:9032 assets.digital.cabinet-office.gov.uk www.google-analytics.com data:"
+
+play {
+  http {
+    router = prod.Routes
+    requestHandler="play.api.http.GlobalSettingsHttpRequestHandler"
+  }
+  modules.enabled+="com.kenshoo.play.metrics.PlayModule"
+  filters {
+    csrf {
+      cookie.secure=true
+      token.sign=false
+      token.name=ERS_CSRF
```

Comment:
I think it needs to be in quotes too. See [https://www.playframework.com/documentation/2.5.x/resources/confs/filters-helpers/reference.conf.](https://www.playframework.com/documentation/2.5.x/resources/confs/filters-helpers/reference.conf).